src/tribler/gui/core_manager.py

```diff
@@ -143,10 +143,12 @@ def stop(self, quit_app_on_core_finished=True):
 
         self.shutting_down = True
         self._logger.info("Stopping Core manager")
-        if self.core_process or self.core_connected:
+        if (self.core_process or self.core_connected) and not self.core_finished:
             self._logger.info("Sending shutdown request to Tribler Core")
             self.events_manager.shutting_down = True
             TriblerNetworkRequest("shutdown", lambda _: None, method="PUT", priority=QNetworkRequest.HighPriority)
+        elif self.should_quit_app_on_core_finished:
+            self.app_manager.quit_application()
```

Comment:
I'm not sure, but Tribler should crash on this line because you have removed the line `self.app_manager: AppManager = parent.app_manager` from FeedbackDialog class.